hack/schemas/testdata/inline-anyof/output.json

```diff
@@ -0,0 +1,92 @@
+{
+  "type": "object",
+  "anyOf": [
+    {
+      "$ref": "#/definitions/SkaffoldPipeline"
+    }
+  ],
+  "$schema": "http://json-schema-org/draft-07/schema#",
+  "definitions": {
+    "InlineOneOfStruct": {
+      "properties": {
+        "field1": {
+          "type": "string",
+          "description": "should be the first choice",
+          "x-intellij-html-description": "should be the first choice"
+        },
+        "field2": {
+          "type": "string",
+          "description": "should be the second choice",
+          "x-intellij-html-description": "should be the second choice"
+        }
+      },
+      "preferredOrder": [
```

Comment:
Since field1 and field2 are "oneOf" is this preferredOrder valid?

Reply:
question asked in #1908, thank you!